index.html

```diff
@@ -0,0 +1,478 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <nav id="navbar">
+      <header>JS Documentation</header>
+      <ul>
+        <li><a name="Introduction" class="nav-link" href="#Introduction">Introduction</a></li>
+        <li>
+          <a name= "What_you_should_already_know" class="nav-link" href="#What_you_should_already_know"
+            >What you should already know</a
+          >
+        </li>
+        <li>
+          <a class="nav-link" href="#JavaScript_and_Java"
+           name = "JavaScript_and_Java" >JavaScript and Java</a
+          >
+        </li>
+        <li><a class="nav-link" href="#Hello_world" name = "Hello_world">Hello world</a></li>
+        <li><a class="nav-link" href="#Variables" name = "Variables">Variables</a></li>
+        <li>
+          <a class="nav-link" href="#Declaring_variables"
+            name = "Declaring_variables">Declaring variables</a
+          >
+        </li>
+        <li><a class="nav-link" href="#Variable_scope" name = "Variable_scope">Variable scope</a></li>
+        <li>
+          <a class="nav-link" href="#Global_variables" name = "Global_variables">Global variables</a>
+        </li>
+        <li><a class="nav-link" href="#Constants" name = "Constants">Constants</a></li>
+        <li><a class="nav-link" href="#Data_types" name = "Data_types">Data types</a></li>
+        <li>
+          <a class="nav-link" href="#if...else_statement"
+           name="if...else_statement" >if...else statement</a
+          >
+        </li>
+        <li><a class="nav-link" href="#while_statement" name = "while_statement">while statement</a></li>
```

Comment:
- [ ] The if...else_statment section doesn't work make sure to use a valid id name.
![image](https://user-images.githubusercontent.com/74744787/157311012-3feeefcd-1792-4670-92c3-1ac2199c9ce6.png)